cdhweb/people/migrations/0026_modeladmin_perms.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 2.2.17 on 2021-02-09 21:15
+
+from django.db import migrations
+
+
+def create_people_permissions(apps, schema_editor):
+    """Set Wagtail user group permissions for non-Page People models.
+
+    By default, Wagtail creates the Moderator and Editor user groups. Users who
+    access the wagtail backend (/cms) must be in one of these groups.
+
+    Models that inherit from Wagtail's `Page` will have permissions set up for
+    these groups, but Django models that can be managed via Wagtail (through
+    the use of `modeladmin`) will not have permissions set up and will not be
+    visible in Wagtail unless permissions are manually added.
+
+    For People, the relevant models are `Title` and `Person`. `Position` and
+    `PersonRelatedLink` are editable by default as inlines on `Person`.
+    """
+
+    # get models and error if any aren't present - they should be since
+    # wagtail's own migrations have already run
+    Group = apps.get_model("auth", "Group")
+    Permission = apps.get_model("auth", "Permission")
+    editors = Group.objects.get(name="Editors")
+    moderators = Group.objects.get(name="Moderators")
+
+    # get permissions to assign
+    add_title = Permission.objects.get(codename="add_title")
+    change_title = Permission.objects.get(codename="change_title")
+    delete_title = Permission.objects.get(codename="delete_title")
+    view_title = Permission.objects.get(codename="view_title")
+    add_person = Permission.objects.get(codename="add_person")
+    change_person = Permission.objects.get(codename="change_person")
+    delete_person = Permission.objects.get(codename="delete_person")
+    view_person = Permission.objects.get(codename="view_person")
+
+    # give moderators all permissions on Title and Person
+    moderators.permissions.set([
+        add_title, change_title, delete_title, view_title,
+        add_person, change_person, delete_person, view_person
+    ])
+
+    # editors have all permissions on Person, but none for Title
+    editors.permissions.set([
+        add_person, change_person, delete_person, view_person
```

Comment:
is view title permission needed to add it to a person, or only for the list view?

Reply:
it's not needed to add it to a person, and removing it means that they don't see the heading for the list view in admin, which i think is what we want